build_tools/kokoro/gcp_ubuntu/cmake/linux/x86-turing/build.sh

```diff
@@ -29,14 +29,66 @@ dpkg -l | grep nvidia
 nvidia-smi || true
 
 ./build_tools/kokoro/gcp_ubuntu/check_vulkan.sh
-
+IREE_SRC_DIR="$PWD"
 # TODO(gcmn): It would be nice to be able to build and test as much as possible,
 # so a build failure only prevents building/testing things that depend on it and
 # we can still run the other tests.
 # TODO: Add "-DIREE_TARGET_BACKEND_CUDA=ON -DIREE_HAL_DRIVER_CUDA=ON" once the
 # VMs have been updated with the correct CUDA SDK.
 echo "Building with cmake"
-./build_tools/cmake/clean_build.sh
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+cd ${ROOT_DIR?}
+rm -rf build/
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+CMAKE_BIN=${CMAKE_BIN:-$(which cmake)}
+
+"$CMAKE_BIN" --version
+ninja --version
+
+cd ${ROOT_DIR?}
+if [ -d "build" ]
+then
+  echo "Build directory already exists. Will use cached results there."
+else
+  echo "Build directory does not already exist. Creating a new one."
+  mkdir build
+fi
+cd build
+
+CMAKE_ARGS=(
+  "-G" "Ninja"
+  # Let's make linking fast
+  "-DIREE_ENABLE_LLD=ON"
+  "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+
+  # Enable docs build on the CI. The additional builds are pretty fast and
+  # give us early warnings for some types of website publication errors.
+  "-DIREE_BUILD_DOCS=ON"
+
+  # Enable building the python bindings on CI. Most heavy targets are gated on
+  # IREE_ENABLE_TENSORFLOW, so what's left here should be fast.
+  "-DIREE_BUILD_PYTHON_BINDINGS=ON"
+
+  # Enable assertions. We wouldn't want to be testing *only* with assertions
```

Comment:
This comment doesn't make sense here. I think having assertions here may be a good idea, but this is specifically referencing the script this was copied from